controllers/nf/smf/resources_test.go

```diff
@@ -188,33 +190,32 @@ func TestCreateConfigMap(t *testing.T) {
 			Name:      "free5gc-upf-1",
 			Namespace: "test-smf-deployment",
 		},
-		Spec: nephiov1alpha1.UPFDeploymentSpec{
-			NFDeploymentSpec: nephiov1alpha1.NFDeploymentSpec{
-				ConfigRefs: []apiv1.ObjectReference{},
-				Capacity: &nephioreqv1alpha1.CapacitySpec{
-					MaxUplinkThroughput:   resource.MustParse("1G"),
-					MaxDownlinkThroughput: resource.MustParse("5G"),
-				},
-				Interfaces: interfaces,
-				NetworkInstances: []nephiov1alpha1.NetworkInstance{
-					{
-						Name: "vpc-internet",
-						Interfaces: []string{
-							"n6",
-						},
-						DataNetworks: []nephiov1alpha1.DataNetwork{
-							{
-								Name: &dnnName,
-								Pool: []nephiov1alpha1.Pool{
-									{
-										Prefix: "100.100.0.0/16",
-									},
+		Spec: nephiov1alpha1.NFDeploymentSpec{
```

Comment:
@VamshiKrishnaNemalikonda I saw this unit test is failing, maybe it's going to be necessary to adapt the output expected.

Reply:
@electrocucaracha  it's fixed now. Thank you